bpf/sockops/sockops_config.h

```diff
@@ -15,5 +15,5 @@
  *  along with this program; if not, write to the Free Software
  *  Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
  */
-#define SOCK_OPS_MAP sock_ops_map
+#define SOCK_OPS_MAP cilium_sock_ops_map
```

Comment:
@nirmoy can you please add the "Fixes: " commit id that this PR is fixing? Something similar to https://github.com/cilium/cilium/commit/0b9c86d271d9ae6de42303565744f604c32b5a5d

Reply:
added "Fixes" and rebased.